charts/psmdb-operator/README.md

```diff
@@ -0,0 +1,51 @@
+# psmdb-operator: A chart for installing the Percona Kubernetes operator for MongoDB
+
+This is an implementation of PSMDB operator Deployment and other stuff could be found here:
+* <https://github.com/percona/percona-server-mongodb-operator>
+
+## Pre Requisites
+* Kubernetes 1.11+
+* PV support on underlying infrastructure (only if you are provisioning persistent volume).
+* Requires at least `v2.4.0` version of helm to support
+
+## Deloyment Details
+* <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>
+
+## Chart Details
+This chart will do the following:
+* Deploy PSMDB operator pod for the further MongoDB creation inside K8S
+
+### Installing the Chart
+To install the chart with the release name `psmdb` using a dedicated namespace(recommended):
+
+```sh
+helm repo add percona https://percona-lab.github.io/percona-helm-charts/
+helm install percona/psmdb-operator --version 0.1.9 --namespace my-namespace
+```
+
+The chart can be customized using the following configurable parameters:
+
+| Parameter                       | Description                                                                   | Default                                   |
+| ------------------------------- | ------------------------------------------------------------------------------| ------------------------------------------|
+| `image.repository`              | PSMDB operator container image name                                           | `percona/percona-server-mongodb-operator` |
+| `image.tag`                     | PSMDB operator container image tag                                            | `1.4.0`                                   |
+| `image.pullPolicy`              | PSMDB operator container pull policy                                          | `Always`                                  |
+| `image.pullSecrets`             | PSMDB operator pod pull secret                                                | `[]`                                      |
+| `replicaCount`                  | PSMDB operator pod quantity                                                   | `1`                                       |
```

Comment:
It's better to start "Pod" and "Operator" words with a capital letter (like official K8S docs are doing)

Reply:
done